### ISSUE TITLE
Vdebug crashes on start-up when debugging node.js

### ISSUE THREAD

**Issue by apolunin** [joonty/vdebug#223]
I installed Vdebug using 'vim plug' package manager.
I installed komodo-debug as Vdebug documentation suggests.
I'm trying to do the following:
* Open my source file in vim.
* Hit F5. Expected message is displayed: "Waiting for a connection (Ctrl-C to cancel, this message will self-destruct in  20  seconds...)".
* Now I try to launch debug session using the following command: node-dbgp -h localhost -p 9000 main.js

The following error is displayed in vim:
```
An error occured: <type 'exceptions.TypeError'>
Traceback (most recent call last):
  File "/Users/andrey/.vim/plugged/vdebug/plugin/python/start_vdebug.py", line 31, in run
    self.runner.run()
  File "/Users/andrey/.vim/plugged/vdebug/plugin/python/vdebug/runner.py", line 160, in run
    self.open()
  File "/Users/andrey/.vim/plugged/vdebug/plugin/python/vdebug/runner.py", line 71, in open
    self.refresh(status)
  File "/Users/andrey/.vim/plugged/vdebug/plugin/python/vdebug/runner.py", line 99, in refresh
    if str(status) == "interactive":
TypeError: __str__ returned non-string (type NoneType)
```
I'm on OS X system using vim compiled by homebrew.
Here is what vim --version displays:

```
VIM - Vi IMproved 7.4 (2013 Aug 10, compiled Aug 26 2015 12:50:11)
MacOS X (unix) version
Included patches: 1-826
Compiled by Homebrew
Huge version without GUI.  Features included (+) or not (-):
+acl             +farsi           +mouse_netterm   +syntax
+arabic          +file_in_path    +mouse_sgr       +tag_binary
+autocmd         +find_in_path    -mouse_sysmouse  +tag_old_static
-balloon_eval    +float           +mouse_urxvt     -tag_any_white
-browse          +folding         +mouse_xterm     -tcl
++builtin_terms  -footer          +multi_byte      +terminfo
+byte_offset     +fork()          +multi_lang      +termresponse
+cindent         -gettext         -mzscheme        +textobjects
-clientserver    -hangul_input    +netbeans_intg   +title
+clipboard       +iconv           +path_extra      -toolbar
+cmdline_compl   +insert_expand   +perl            +user_commands
+cmdline_hist    +jumplist        +persistent_undo +vertsplit
+cmdline_info    +keymap          +postscript      +virtualedit
+comments        +langmap         +printer         +visual
+conceal         +libcall         +profile         +visualextra
+cryptv          +linebreak       +python          +viminfo
+cscope          +lispindent      -python3         +vreplace
+cursorbind      +listcmds        +quickfix        +wildignore
+cursorshape     +localmap        +reltime         +wildmenu
+dialog_con      -lua             +rightleft       +windows
+diff            +menu            +ruby            +writebackup
+digraphs        +mksession       +scrollbind      -X11
-dnd             +modify_fname    +signs           -xfontset
-ebcdic          +mouse           +smartindent     -xim
+emacs_tags      -mouseshape      -sniff           -xsmp
+eval            +mouse_dec       +startuptime     -xterm_clipboard
+ex_extra        -mouse_gpm       +statusline      -xterm_save
+extra_search    -mouse_jsbterm   -sun_workshop    -xpm
   system vimrc file: "$VIM/vimrc"
     user vimrc file: "$HOME/.vimrc"
 2nd user vimrc file: "~/.vim/vimrc"
      user exrc file: "$HOME/.exrc"
  fall-back for $VIM: "/usr/local/share/vim"
Compilation: /usr/bin/clang -c -I. -Iproto -DHAVE_CONFIG_H   -F/usr/local/Frameworks -DMACOS_X_UNIX  -Os -w -pipe -march=native -mmacosx-version-min=10.10 -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1       
Linking: /usr/bin/clang   -L. -L/usr/local/lib -L/usr/local/lib -F/usr/local/Frameworks -Wl,-headerpad_max_install_names -o vim        -lm  -lncurses -liconv -framework Cocoa   -fstack-protector  -L/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE -lperl -F/usr/local/Cellar/python/2.7.10_2/Frameworks -framework Python   -lruby.2.0.0 -lobjc 
```

**Comment by joonty** [comment]
Hi there,

Which version of node are you using? I've recently discovered that komodo-debug only works with versions less than or equal to nodejs v0.10.

If you do have 0.10 or less, can you try using the `dev` branch of Vdebug? I'm about to create a new release this week, and there are a lot of fixes since the previous release (which was last year).

Thanks

**Comment by apolunin** [comment]
Ah, this might be the issue... I'm using the latest node (currently 0.12.7).
Will there be a way to use Vdebug with later versions?

**Comment by joonty** [comment]
That's a great question!

Did some digging and found that Activestate are [planning on supporting newer nodejs versions with Komodo 9](http://forum.komodoide.com/t/node-js-debugger-never-finishes-starting/1321/2), and it's currently on 8. So hopefully that'll arrive soon. In the mean time are you able to downgrade? Or, probably better, to have the two versions available and use 0.10 for debugging?

I'll use this issue to add this to the documentation for setting up node JS.

Thanks

**Comment by apolunin** [comment]
Yep, I can downgrade and use earlier version. I hope future version of komodo-debug will add support for fresh node version.

@joonty , thank you for the help and quick reply!
